src/shared/utils/upload-img.util.ts

```diff
@@ -0,0 +1,20 @@
+import configuration from "~shared/config/configuration";
+import * as cloudinary from 'cloudinary';
+import DataURIParser = require('datauri/parser');
+
+export const uploadImg = async (file: any, folderName: string, uniqId?: string): Promise<string> => {
+
+    const { cloudinary: config } = configuration();
+    cloudinary.v2.config({
+        cloud_name: config.name,
+        api_key: config.key,
+        api_secret: config.secret
```

Comment:
Codacy found an issue: [Identifier 'api_secret' is not in camel case.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=5837760)